rate_counter/static/rate/css/base.css

```diff
@@ -17,4 +17,19 @@ th {
 .chart_div {
     width: 900px;
     height: 500px;
-}
\ No newline at end of file
+}
+
+.col-sm-5 {
```

Comment:
here you override Bootstrap built in css class. It is not recommended to do.
Html elements work with multiple classes.
For example if you have:
`<div class="col-sm-5">`
and you want to change style of it I recommend to do it like
`<div class="col-sm-5 my_custom_class">`
so you will not brake Bootstrap styles.